lib/evmone/analysis.cpp

```diff
@@ -15,6 +15,18 @@ bool is_terminator(uint8_t c) noexcept
     return c == OP_JUMP || c == OP_JUMPI || c == OP_STOP || c == OP_RETURN || c == OP_REVERT ||
            c == OP_SELFDESTRUCT;
 }
+
+int load_1byte_immediate_value(const uint8_t* code, size_t code_size, size_t& index) noexcept
```

Comment:
Minor stylistic suggestion: I would leave updating of `index` on the calling side, i.e. pass it by value here. Then it would be easier to understand what's happending with it, when you look the calling side

Reply:
Yeah, but we update the index conditionally. We can rename `load_` into `consume_` though.

Reply:
Reworked.